.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - alpha
+      - beta
+      - stable
+      - mrbeam2-beta
+      - mrbeam2-stable
```

Comment:
we don't need these anymore, when this change comes to stable we are not using mrbeam2-* branches anymore

Reply:
Removed